iterations.r

```diff
@@ -31,6 +31,20 @@ stations_metadata <-
     .url = configs$vegvesen_url
     ) 
 
+#verify the lenght of the list
+length(stations_metadata)
+length(stations_metadata[[1]])
+stations_metadata[[1]][[1]] 
+
+transform_metadata_to_df <- function(stations_metadata) {
```

Comment:
You should not include this function in the iterations.r script. 

The function source("functions/data_transformations.r") loads the function if it is written in the data_transformations.r script.